languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -53893,8 +53937,9 @@ though, and has since been slightly modified)-->
                 <example>Peut-être l'<marker>apprécie-t-il aussi</marker>.</example>
                 <example>Les <marker>saviez-vous aussi</marker> agiles ?</example>
                 <example>On peut commencer <marker>par elle aussi</marker>.</example>
+                <example>C'est <marker> elle aussi</marker>qui habite le coeur des mères.</example>
```

Comment:
It's odd that there are 2 spaces and that the 2nd space is in between <marker>…</marker>